### PR TITLE
Create Event

### PR DESCRIPTION
**Pull request by lfallert**
This feature should work alongside the Client Side to pull information from the form submitted to store in the database. To test, make sure that after event details are submitted, that they are stored in the database.

### CODE REVIEW

**Review by CNichole726 — APPROVED** [review]
works on my machine

File: src/main/java/com/launchcode/AroundTownServer/models/Event.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
+package com.launchcode.AroundTownServer.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.util.Objects;
+
+
+//import javax.validation.constraints.NotNull;
```
Comment (miketully426) [inline src/main/java/com/launchcode/AroundTownServer/models/Event.java:9]: Do we need this commented code?

File: src/main/java/com/launchcode/AroundTownServer/models/Event.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        this.name = name;
+        this.description = description;
+        this.location = location;
+        this.date = date;
+        this.time = time;
+        this.entryCost = entryCost;
+        this.familyFriendly = familyFriendly;
+    }
+
+    public Event() {
```
Comment (miketully426) [inline src/main/java/com/launchcode/AroundTownServer/models/Event.java:43]: By convention the empty constructor should be before the all arguments constructor.

File: src/main/java/com/launchcode/AroundTownServer/models/Event.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    }
+
+    public String getName() {
```
Comment (miketully426) [inline src/main/java/com/launchcode/AroundTownServer/models/Event.java:46]: Can we use Lombok and get rid of all these getters and setters?

**Review by miketully426 — CHANGES_REQUESTED** [review]

File: src/main/java/com/launchcode/AroundTownServer/models/Event.java (hunk 3 above)
Comment (lfallert) [reply src/main/java/com/launchcode/AroundTownServer/models/Event.java:46]: Sure thing!

**Pull request closed without merging**